sound/soc/sof/intel/hda-loader.c

```diff
@@ -375,7 +375,6 @@ int hda_dsp_cl_boot_firmware(struct snd_sof_dev *sdev)
 
 static int cl_dsp_init_skl(struct snd_sof_dev *sdev)
 {
-	const struct sof_intel_dsp_desc *chip = sdev->hda->desc;
```

Comment:
I believe you haven't tested(or even compile) your update? removing 'chip' here doesn't compile?